### PR TITLE
CSS update: overview of shapes guide

### PR DESCRIPTION
**Pull request by estelle**
Brought this page up to date.

### CODE REVIEW

File: files/en-us/web/css/css_shapes/overview_of_shapes/index.md [hunk 1]
```diff
@@ -90,26 +94,24 @@ In the example below we have added a `shape-margin` to a basic shape. Change the
...
 
-The Shape Path Editor will be enabled by default in Firefox 60 for shapes generated via `clip-path`. You can also use it to edit shapes generated via `shape-outside`, but only when you enable it via the `layout.css.shape-outside.enabled` pref.
+## More CSS Shapes Features
 
-## Future CSS Shapes Features
-
-The initial Shapes specification included a property `shape-inside` for creating shapes inside an element. This property, along with the possibility of creating shapes on non-floated elements, has been moved to [level 2](https://drafts.csswg.org/css-shapes-2/) of the specification. As the `shape-inside` property was initially in Level 1 of the specification, you may find tutorials on the web detailing both properties.
+In this guide, we discussed wrapping text around floated shapes. See the [CSS shapes module](/en-US/docs/Web/CSS/CSS_shapes) for links all the module and related features, including all the shape functions and additional guides.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/css_shapes/overview_of_shapes/index.md:117]:
```suggestion
In this guide, we discussed wrapping text around floated shapes. See the [CSS shapes module](/en-US/docs/Web/CSS/CSS_shapes) for links to all the module features plus additional related features. This includes all the shape functions and relevant guides.
```

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Nice work @estelle. I just had a few text suggestions for you; nothing major.

**Comment by estelle** [comment]
@chrisdavidmills Thanks for the quick turnaround. I committed all your suggestions.

**Review by chrisdavidmills — APPROVED** [review]
@estelle fabulous. Let's get this merged then. Thanks for the awesome work.

**Pull request merged**